src/realm/link_view.cpp

```diff
@@ -41,10 +49,19 @@ void LinkView::generate_patch(const ConstLinkViewRef& ref, std::unique_ptr<Hando
 LinkViewRef LinkView::create_from_and_consume_patch(std::unique_ptr<HandoverPatch>& patch, Group& group)
 {
     if (patch) {
-        TableRef tr = Table::create_from_and_consume_patch(patch->m_table, group);
-        LinkViewRef result = tr->get_linklist(patch->m_col_num, patch->m_row_ndx);
-        patch.reset();
-        return result;
+        if (patch->m_table) {
+            TableRef tr = Table::create_from_and_consume_patch(patch->m_table, group);
+            LinkViewRef result = tr->get_linklist(patch->m_col_num, patch->m_row_ndx);
+            patch.reset();
+            return result;
+        }
+        else {
+            // We end up here if we're handing over a detached linkview.
```

Comment:
I seem to have a bad habit. It's like for a class SomeClass, my brain infers that there is a concept behind the class, which is then called someclass. I'll fix it. Both the code and the habit, I hope.